action_server/src/sema4ai/action_server/_ask_user.py

```diff
@@ -0,0 +1,106 @@
+import typing
+from logging import getLogger
+
+from sema4ai.action_server.package._package_publish import Organization
+
+log = getLogger(__name__)
+
+
+def ask_user_input_to_proceed(msg: str) -> bool:
+    """
```

Comment:
Might be good to start each docstring with a summary line. (our [convention](https://www.notion.so/robocorptechnologies/Docstrings-c41098ba80cf4ba79036562aba6772b7?pvs=4) for docstrings)

At the end, the linter for them should pass nicely. (Invoke command)

Reply:
This was actually old code, the file was just moved to more generic place. I have followed the convention that was used in this repository. I'm not sure if it's a good idea to start using other convention when adding new docstrings, may be just more confusing.

Reply:
For internal functions, I think it's valid to skip the summary if it'll mostly duplicate what's already in the signature (if it's obvious).

If something isn't obvious, then a summary should be added with that...

Additional note: for things which are part of the public API then it's not the same. All public API functions should have a summary with more details on what the user should expect (and even a change only in the docs could become a breaking change).

Reply:
Fair point, when we'll have that auto-formatter in place @OvidiuCode was thinking about, we'd be able to reformat everything so we'd inline the entire codebase. If the current linting isn't failing, then the format is considered accepted.